APIs_Drivers/BufferedSerial_echo/main.cpp

```diff
@@ -0,0 +1,36 @@
+#include "mbed.h"
+
+
+// Maximum number of element the application buffer can contain
+#define MAXIMUM_BUFFER_SIZE                                                  32
+
+
+// Create a DigitalOutput object to toggle an LED whenever data is received.
+static DigitalOut led(LED1);
+
+// Create a new UARTSerial object with a default baud rate.
+static BufferedSerial serial_port(USBTX, USBRX);
+
+
+int main(void) {
```

Comment:
Could you remove a few too many empty lines?

Reply:
Done on purpose because I have been following Black style
https://black.readthedocs.io/en/stable/the_black_code_style.html#empty-lines
But Ok.

Reply:
[Done](https://github.com/ARMmbed/mbed-os-examples-docs_only/compare/dd81220d2f1c723333609afa51c1479d12251021..17b7244223e0208efdc12274754ea22467bc8234).